src/components/UI/Charts/Sunburst/sunburst-d3.jsx

```diff
@@ -265,6 +265,25 @@ const sunburstD3 = (ref, data, config, getSelectedPhenotype, formatters) => {
     const textData = p || selectedPhenotype;
     centerText.text(() => centerTextFormatter(textData.data)).call(wrap);
   };
+
+  const findNodeByKey = (key, node, returnNode) => {
```

Comment:
can this function be extracted on it's own? 

instead of looping through all nodes, could "some" or "find" do the job?